pyproject.toml

```diff
@@ -179,7 +179,9 @@ select = [
     # print statements
     "T201",
     # pandas-vet
-    "PD"
+    "PD",
+    # numpy 2.0
+    "NPY201"
```

Comment:
what is this?

Reply:
- Let's `ruff` check for old numpy usage:
  - https://docs.astral.sh/ruff/rules/numpy2-deprecation/

Reply:
Hi @amontanez24, exactly; on CTGAN, there were no changes (no new linting errors), but I found 6 NPY201 linting errors on SDV. Let me know if we should keep it here.

Reply:
Ok, we can keep